src/pages/home/HomePage.tsx

```
@@ -26,53 +29,76 @@ interface HomeExpenseItemType {
+      <Divider />
+      <SettlementList />
+    </Flex>
   );
+}
+
+function MainHeader() {
   const navigate = useNavigate();
   const theme = useTheme();
+  return (
+    <Header
+        type="TitleCenter"
```

ESLint

[error] Delete `··`

(prettier/prettier)